clang/include/clang/CIR/Dialect/IR/CIRAttrs.td

```diff
@@ -521,4 +521,33 @@ def GlobalCtorAttr : CIR_Attr<"GlobalCtor", "globalCtor"> {
   ];
   let skipDefaultBuilders = 1;
 }
+
+def BitfieldInfoAttr : CIR_Attr<"BitfieldInfo", "bitfieldInfo"> {
+  let parameters = (ins "StringAttr":$name,
+                        "Type":$eltType,
+                        "uint64_t":$storageSize,
+                        "uint64_t":$size,
+                        "uint64_t":$offset,
+                        "bool":$isSigned);
+
+  let assemblyFormat = [{ `<`
+  `(` `name` `=` $name `)`
+  `(` `size` $size `)`
+  `(` `storage` $eltType $storageSize `)`
+  `(` `offset` $offset  `)`
+  `(` `signed` `=` $isSigned `)` `>` }];
```

Comment:
Example of how it should something like:
```
#cir.bitfielf_info<name = "a", size = 7, ...>
```

Also, please add an alias for it in `AliasResult getAlias(Attribute attr, raw_ostream &os)` in `CIRDialect.cpp` so that operations can refer to an easy `#bitfield_fileXYZ`